dashboard/modules/reporter/reporter_agent.py

```diff
@@ -32,10 +33,11 @@
     import gpustat.core as gpustat
 except (ModuleNotFoundError, ImportError):
     gpustat = None
-    logger.warning("`gpustat` package is not installed. GPU monitoring is "
-                   "not available. In Ray 1.4+, the Ray CLI, autoscaler, and "
-                   "dashboard will only be usable via `pip install 'ray["
-                   "default]'`. Please update your install command")
+    if log_once("gpustat_import_warning"):
+        logger.warning("`gpustat` package is not installed. GPU monitoring is "
```

Comment:
would prefer `warnings.warn` but not a big deal.

Reply:
changed